docs/backup-restore.md

```diff
@@ -1,134 +1,183 @@
-# A Guide on How to Backup and Restore Databases Used Through ScalarDB
+# How to Back Up and Restore Databases Used Through ScalarDB
 
-Since ScalarDB provides transaction capability on top of non-transactional (possibly transactional) databases non-invasively, you need to take special care of backing up and restoring the databases in a transactionally-consistent way.
-This document sets out some guidelines for backing up and restoring the databases that ScalarDB supports.
+Since ScalarDB provides transaction capabilities on top of non-transactional or transactional databases non-invasively, you need to take special care to back up and restore the databases in a transactionally consistent way.
 
-## Create Backup
+This guide describes how to back up and restore the databases that ScalarDB supports.
 
-The way to create a backup varies on what database you use and whether or not you use multiple databases.
+## Create a backup
 
+How you create a backup depends on which database you're using and whether or not you're using multiple databases. The following decision tree shows which approach you should take.
 
-The following is the decision tree to decide what approach you should take.
+```mermaid
```

Comment:
I think we are missing an arrow from `Does the database have transaction support?` to `Perform back up with explicit pausing` for the `No` case.

Reply:
Good catch, thank you! I've added a `No` arrow for that part.